docs/configuration.rst

```diff
@@ -986,6 +986,134 @@ remain available as described at the start of the :ref:`Configuration` section.
     Name of the :term:`Provider` used for login. This represents the identifier that is set to define how to
     differentiate between a local sign-in procedure and a dispatched one some known :ref:`authn_providers`.
 
+Network Mode Settings
+~~~~~~~~~~~~~~~~~~~~~
+
+The following configuration parameters are related to Magpie's "Network Mode" which allows networked instances of Magpie
+to authenticate users for each other. All variables defined in this section are only used if
+:envvar:`MAGPIE_NETWORK_ENABLED` is enabled.
+
+.. envvar:: MAGPIE_NETWORK_ENABLED
+
+    [:class:`bool`]
+    (Default: ``False``)
+
+    .. versionadded:: 3.38
+
+    Enable "Network Mode" which enables all functionality to authenticate users using other Magpie instances as
+    external authentication providers.
+
+.. envvar:: MAGPIE_NETWORK_INSTANCE_NAME
+
+    [:class:`str`]
+
+    .. versionadded:: 3.38
+
+    The name of this Magpie instance in the network. This variable is used to determine if an authentication token was
+    issued by this instance of Magpie, or another instance in the network.
+
+    This variable is required if :envvar:`MAGPIE_NETWORK_ENABLED` is ``True``.
+
+.. envvar:: MAGPIE_NETWORK_DEFAULT_TOKEN_EXPIRY
+
+    [:class:`int`]
+    (Default: ``86400``)
+
+    .. versionadded:: 3.38
+
+    The default expiry time (in seconds) for an access token issued for the purpose of network authentication.
+
+.. envvar:: MAGPIE_NETWORK_INTERNAL_TOKEN_EXPIRY
+
+    [:class:`int`]
+    (Default: ``30``)
+
+    .. versionadded:: 3.38
+
+    The default expiry time (in seconds) for an JSON Web Token issued for the purpose of communication between nodes in
+    the network.
+
+.. envvar:: MAGPIE_NETWORK_TOKEN_NAME
+
+    [|constant|_]
+    (Value: ``"magpie_token"``)
+
+    .. versionadded:: 3.38
+
+    The name of the request parameter key whose value is the authentication token issued for the purpose of network
+    authentication.
+
+.. envvar:: MAGPIE_NETWORK_PROVIDER
+
+    [|constant|_]
+    (Value: ``"magpie_network"``)
+
+    .. versionadded:: 3.38
+
+    The name of the external provider that authenticates users using other Magpie instances as external authentication
+    providers.
+
+.. envvar:: MAGPIE_NETWORK_NAME_PREFIX
+
+    [|constant|_]
+    (Value: ``"anonymous_network_"``)
```

Comment:
I'm not sure to understand why `anonymous` is used here specifically and not referenced in the variable name. Is it only for this special user/group? If so, it would be better to have explicit `MAGPIE_NETWORK_ANONYMOUS_NAME_PREFIX`. It would also be better to use `magpie_network_anonymous_` to align with other `magpie_network...` names.